### ISSUE TITLE
pyscenic aucell command does not complete

### ISSUE THREAD

**Issue by leeanapeters** [aertslab/pySCENIC#510]
Hi, thank you for the great tool!

I have managed to run the inital steps of the workflow but am not able to run aucell as the job is killed regardless of if it is through the command line ( with a range of --num_workers), a slurm script, or in a jupyter notebook. 

There is no error other than "killed" output so I am not sure how to troubleshoot. Any help would be appreciated!

Command run:

 pyscenic aucell pln_pbmc_filtered_scenic.loom  reg.csv --output pyscenic_output_1.loom --num_workers 2

Error encountered:
Killed

- pySCENIC version: 0.12.0
- Installation method: conda environment with pip install
- Run environment: CLI, HPC script, jupyter

absl-py==1.0.0
aiohttp==3.8.4
aiosignal==1.3.1
anndata==0.9.2
anyio @ file:///home/conda/feedstock_root/build_artifacts/anyio_1693488585952/work
anytree==2.8.0
arboreto==0.1.6
argon2-cffi @ file:///home/conda/feedstock_root/build_artifacts/argon2-cffi_1692818318753/work
argon2-cffi-bindings @ file:///home/conda/feedstock_root/build_artifacts/argon2-cffi-bindings_1695386548039/work
arrow @ file:///home/conda/feedstock_root/build_artifacts/arrow_1662382474514/work
asttokens @ file:///home/conda/feedstock_root/build_artifacts/asttokens_1694046349000/work
astunparse==1.6.3
async-lru @ file:///home/conda/feedstock_root/build_artifacts/async-lru_1690563019058/work
async-timeout==4.0.2
attrs @ file:///home/conda/feedstock_root/build_artifacts/attrs_1683424013410/work
autodocsumm==0.2.8
Babel @ file:///home/conda/feedstock_root/build_artifacts/babel_1677767029043/work
backcall @ file:///home/conda/feedstock_root/build_artifacts/backcall_1592338393461/work
backports.functools-lru-cache @ file:///home/conda/feedstock_root/build_artifacts/backports.functools_lru_cache_1687772187254/work
beautifulsoup4 @ file:///home/conda/feedstock_root/build_artifacts/beautifulsoup4_1680888073205/work
bleach @ file:///home/conda/feedstock_root/build_artifacts/bleach_1674535352125/work
bokeh==2.4.3
boltons==23.0.0
Brotli @ file:///home/conda/feedstock_root/build_artifacts/brotli-split_1693583441880/work
cached-property @ file:///home/conda/feedstock_root/build_artifacts/cached_property_1615209429212/work
cachetools==5.3.0
certifi==2023.7.22
cffi @ file:///home/conda/feedstock_root/build_artifacts/cffi_1695367044379/work
charset-normalizer @ file:///home/conda/feedstock_root/build_artifacts/charset-normalizer_1688813409104/work
click==8.1.3
cloudpickle==2.2.1
comm @ file:///home/conda/feedstock_root/build_artifacts/comm_1691044910542/work
contourpy==1.1.1
ctxcore==0.2.0
cycler==0.12.0
cytoolz @ file:///croot/cytoolz_1667465931118/work
cytopy==2.0.1
dask==2023.5.0
dcor==0.5.6
debugpy @ file:///home/conda/feedstock_root/build_artifacts/debugpy_1695534280282/work
decorator @ file:///home/conda/feedstock_root/build_artifacts/decorator_1641555617451/work
defusedxml @ file:///home/conda/feedstock_root/build_artifacts/defusedxml_1615232257335/work
detecta==0.0.5
dill==0.3.6
distributed==2023.5.0
dm-tree==0.1.7
entrypoints @ file:///home/conda/feedstock_root/build_artifacts/entrypoints_1643888246732/work
exceptiongroup @ file:///home/conda/feedstock_root/build_artifacts/exceptiongroup_1692026125334/work
executing @ file:///home/conda/feedstock_root/build_artifacts/executing_1667317341051/work
fastjsonschema @ file:///home/conda/feedstock_root/build_artifacts/python-fastjsonschema_1690055433477/work/dist
fdasrsf==2.3.10
findiff==0.8.9
flatbuffers==1.12
FlowIO==0.9.5
FlowUtils==0.9.1
fonttools==4.33.3
fqdn @ file:///home/conda/feedstock_root/build_artifacts/fqdn_1638810296540/work/dist
frozendict==2.3.5
frozenlist==1.3.3
fsspec==2023.3.0
gast==0.4.0
google-auth==2.16.2
google-auth-oauthlib==0.4.6
google-pasta==0.2.0
gpflow==2.5.2
GPy==1.10.0
graphtools==1.5.2
graphviz==0.14.2
grpcio==1.51.3
h5py==3.8.0
harmonypy==0.0.5
HeapDict==1.0.1
idna @ file:///home/conda/feedstock_root/build_artifacts/idna_1663625384323/work
imagesize==1.3.0
imbalanced-learn==0.7.0
importlib-metadata @ file:///home/conda/feedstock_root/build_artifacts/importlib-metadata_1688754491823/work
importlib-resources @ file:///home/conda/feedstock_root/build_artifacts/importlib_resources_1695414790617/work
interlap==0.2.7
ipykernel @ file:///home/conda/feedstock_root/build_artifacts/ipykernel_1693880262622/work
ipython==7.18.1
isoduration @ file:///home/conda/feedstock_root/build_artifacts/isoduration_1638811571363/work/dist
jedi @ file:///home/conda/feedstock_root/build_artifacts/jedi_1690896916983/work
Jinja2 @ file:///home/conda/feedstock_root/build_artifacts/jinja2_1654302431367/work
joblib==1.2.0
json5 @ file:///home/conda/feedstock_root/build_artifacts/json5_1688248289187/work
jsonpointer @ file:///home/conda/feedstock_root/build_artifacts/jsonpointer_1695397259927/work
jsonschema @ file:///home/conda/feedstock_root/build_artifacts/jsonschema-meta_1695228989494/work
jsonschema-specifications @ file:///home/conda/feedstock_root/build_artifacts/jsonschema-specifications_1689701150890/work
jupyter-events @ file:///home/conda/feedstock_root/build_artifacts/jupyter_events_1691505939576/work
jupyter-lsp @ file:///home/conda/feedstock_root/build_artifacts/jupyter-lsp-meta_1685453365113/work/jupyter-lsp
jupyter_client @ file:///home/conda/feedstock_root/build_artifacts/jupyter_client_1693317508789/work
jupyter_core @ file:///home/conda/feedstock_root/build_artifacts/jupyter_core_1695648223821/work
jupyter_server @ file:///home/conda/feedstock_root/build_artifacts/jupyter_server_1693487358826/work
jupyter_server_terminals @ file:///home/conda/feedstock_root/build_artifacts/jupyter_server_terminals_1673491454549/work
jupyterlab @ file:///home/conda/feedstock_root/build_artifacts/jupyterlab_1694728214446/work
jupyterlab-pygments @ file:///home/conda/feedstock_root/build_artifacts/jupyterlab_pygments_1649936611996/work
jupyterlab_server @ file:///home/conda/feedstock_root/build_artifacts/jupyterlab_server-split_1694531968623/work
KDEpy==1.0.10
keras==2.9.0
Keras-Preprocessing==1.1.2
kiwisolver==1.4.5
lark==1.1.2
lazy_loader==0.1rc2
leidenalg==0.8.10
libclang==14.0.1
littleutils==0.2.2
llvmlite==0.41.0
locket==1.0.0
loompy==3.0.7
lz4==4.3.2
Markdown==3.4.2
MarkupSafe @ file:///home/conda/feedstock_root/build_artifacts/markupsafe_1695367437975/work
matplotlib==3.6.3
matplotlib-inline @ file:///home/conda/feedstock_root/build_artifacts/matplotlib-inline_1660814786464/work
MiniSom==2.3.0
mistune @ file:///home/conda/feedstock_root/build_artifacts/mistune_1692116650819/work
mongoengine==0.20.0
mpldatacursor==0.7.1
msgpack==1.0.5
multidict==6.0.4
multimethod==1.8
multipledispatch==0.6.0
multiprocessing-on-dill==3.5.0a4
natsort==8.4.0
nbclient @ file:///home/conda/feedstock_root/build_artifacts/nbclient_1684790896106/work
nbconvert @ file:///home/conda/feedstock_root/build_artifacts/nbconvert-meta_1693331710275/work
nbformat @ file:///home/conda/feedstock_root/build_artifacts/nbformat_1690814868471/work
nest-asyncio @ file:///home/conda/feedstock_root/build_artifacts/nest-asyncio_1664684991461/work
networkx==3.0
notebook_shim @ file:///home/conda/feedstock_root/build_artifacts/notebook-shim_1682360583588/work
numba==0.58.0
numexpr==2.8.4
numpy==1.22.0
numpy-groupies==0.9.20
oauthlib==3.2.2
openpyxl==3.0.6
opt-einsum==3.3.0
outdated==0.2.1
overrides @ file:///home/conda/feedstock_root/build_artifacts/overrides_1691338815398/work
packaging @ file:///home/conda/feedstock_root/build_artifacts/packaging_1681337016113/work
pandas==1.3.5
pandas-flavor==0.3.0
pandocfilters @ file:///home/conda/feedstock_root/build_artifacts/pandocfilters_1631603243851/work
paramz==0.9.5
parso @ file:///home/conda/feedstock_root/build_artifacts/parso_1638334955874/work
partd==1.3.0
pexpect @ file:///home/conda/feedstock_root/build_artifacts/pexpect_1667297516076/work
phate==1.0.7
PhenoGraph==1.5.7
pickleshare @ file:///home/conda/feedstock_root/build_artifacts/pickleshare_1602536217715/work
Pillow==9.4.0
pingouin==0.3.12
pkgutil_resolve_name @ file:///home/conda/feedstock_root/build_artifacts/pkgutil-resolve-name_1694617248815/work
platformdirs @ file:///home/conda/feedstock_root/build_artifacts/platformdirs_1690813113769/work
prometheus-client @ file:///home/conda/feedstock_root/build_artifacts/prometheus_client_1689032443210/work
prompt-toolkit @ file:///home/conda/feedstock_root/build_artifacts/prompt-toolkit_1688565951714/work
protobuf==3.19.6
psutil @ file:///home/conda/feedstock_root/build_artifacts/psutil_1695367190297/work
ptyprocess @ file:///home/conda/feedstock_root/build_artifacts/ptyprocess_1609419310487/work/dist/ptyprocess-0.7.0-py2.py3-none-any.whl
pure-eval @ file:///home/conda/feedstock_root/build_artifacts/pure_eval_1642875951954/work
pyarrow==11.0.0
pyasn1==0.4.8
pyasn1-modules==0.2.8
pycparser @ file:///home/conda/feedstock_root/build_artifacts/pycparser_1636257122734/work
Pygments @ file:///home/conda/feedstock_root/build_artifacts/pygments_1691408637400/work
PyGSP==0.5.1
pymongo==3.12.3
pyparsing==3.1.1
pyscenic==0.12.0
PySocks @ file:///home/conda/feedstock_root/build_artifacts/pysocks_1661604839144/work
python-dateutil @ file:///home/conda/feedstock_root/build_artifacts/python-dateutil_1626286286081/work
python-json-logger @ file:///home/conda/feedstock_root/build_artifacts/python-json-logger_1677079630776/work
pytz @ file:///home/conda/feedstock_root/build_artifacts/pytz_1693930252784/work
PyYAML @ file:///home/conda/feedstock_root/build_artifacts/pyyaml_1695373436676/work
pyzmq @ file:///home/conda/feedstock_root/build_artifacts/pyzmq_1695384324905/work
rdata==0.8
referencing @ file:///home/conda/feedstock_root/build_artifacts/referencing_1691337268233/work
requests @ file:///home/conda/feedstock_root/build_artifacts/requests_1684774241324/work
requests-oauthlib==1.3.1
rfc3339-validator @ file:///home/conda/feedstock_root/build_artifacts/rfc3339-validator_1638811747357/work
rfc3986-validator @ file:///home/conda/feedstock_root/build_artifacts/rfc3986-validator_1598024191506/work
rpds-py @ file:///home/conda/feedstock_root/build_artifacts/rpds-py_1694630943819/work
rsa==4.9
s-gd2==1.8
scanpy==1.9.1
scikit-datasets==0.2.0
scikit-fda==0.5
scikit-learn==0.24.2
scipy==1.6.0
scprep==1.2.0
seaborn==0.13.0
Send2Trash @ file:///home/conda/feedstock_root/build_artifacts/send2trash_1682601222253/work
session-info==1.0.0
shap==0.38.1
Shapely==1.7.1
six @ file:///home/conda/feedstock_root/build_artifacts/six_1620240208055/work
sniffio @ file:///home/conda/feedstock_root/build_artifacts/sniffio_1662051266223/work
sortedcontainers==2.4.0
soupsieve @ file:///home/conda/feedstock_root/build_artifacts/soupsieve_1693929250441/work
sphinxcontrib-applehelp==1.0.2
sphinxcontrib-devhelp==1.0.2
sphinxcontrib-jsmath==1.0.1
sphinxcontrib-qthelp==1.0.3
stack-data @ file:///home/conda/feedstock_root/build_artifacts/stack_data_1669632077133/work
statsmodels==0.12.2
stdlib-list==0.8.0
tables==3.7.0
tasklogger==1.1.2
tblib==1.7.0
tensorboard==2.9.0
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.8.1
tensorflow==2.9.1
tensorflow-estimator==2.9.0
tensorflow-io-gcs-filesystem==0.26.0
tensorflow-probability==0.16.0
termcolor==1.1.0
terminado @ file:///home/conda/feedstock_root/build_artifacts/terminado_1670253674810/work
threadpoolctl==3.1.0
tinycss2 @ file:///home/conda/feedstock_root/build_artifacts/tinycss2_1666100256010/work
tomli @ file:///home/conda/feedstock_root/build_artifacts/tomli_1644342247877/work
toolz @ file:///croot/toolz_1667464077321/work
tornado @ file:///home/conda/feedstock_root/build_artifacts/tornado_1695373547159/work
tqdm==4.65.0
traitlets @ file:///home/conda/feedstock_root/build_artifacts/traitlets_1694710374095/work
typing-utils @ file:///home/conda/feedstock_root/build_artifacts/typing_utils_1622899189314/work
typing_extensions @ file:///home/conda/feedstock_root/build_artifacts/typing_extensions_1695040754690/work
umap-learn==0.4.6
uri-template @ file:///home/conda/feedstock_root/build_artifacts/uri-template_1688655812972/work/dist
urllib3 @ file:///home/conda/feedstock_root/build_artifacts/urllib3_1695228328709/work
wcwidth @ file:///home/conda/feedstock_root/build_artifacts/wcwidth_1673864653149/work
webcolors @ file:///home/conda/feedstock_root/build_artifacts/webcolors_1679900785843/work
webencodings @ file:///home/conda/feedstock_root/build_artifacts/webencodings_1694681268211/work
websocket-client @ file:///home/conda/feedstock_root/build_artifacts/websocket-client_1694440396544/work
Werkzeug==2.1.2
wrapt==1.15.0
xgboost==1.6.1
xlrd==1.2.0
yarl==1.8.2
yellowbrick==1.2
zict==2.2.0
zipp @ file:///home/conda/feedstock_root/build_artifacts/zipp_1695255097490/work

**Comment by ghuls** [comment]
Are you sure that you are not just running out of job time on your HPC cluster?
